funcx/endpoint/endpoint.py

```diff
@@ -161,38 +166,34 @@ def register_with_hub(endpoint_uuid, endpoint_dir, address, redis_host='funcx-re
     return r.json()
 
 
-def configure_endpoint(args, config_file=None, global_config=None):
+@app.command(name="configure", help="Configure an endpoint")
+def configure_endpoint(
+        name: str = typer.Argument("default", help="endpoint name", autocompletion=complete_endpoint_name),
+        endpoint_config: str = typer.Option(None, "--endpoint-config", help="endpoint config file")
+):
     """Configure an endpoint
 
     Drops a config.py template into the funcx configs directory.
     The template usually goes to ~/.funcx/<ENDPOINT_NAME>/config.py
-
-    Parameters
-    ----------
-    args : args object
-       Args object from the arg parsing
-
-    config_file : str
-       Path to a config file to be used instead of the funcX default config file
-
-    global_config : dict
-       Global config dict
     """
-
-    endpoint_dir = os.path.join(args.config_dir, args.name)
-    endpoint_json = os.path.join(endpoint_dir, 'endpoint.json')
+    endpoint_dir = os.path.join(State.FUNCX_DIR, name)
+    new_config_file = os.path.join(endpoint_dir, FUNCX_CONFIG_FILE_NAME)
 
     if not os.path.exists(endpoint_dir):
-        init_endpoint_dir(args.config_dir, args.name, config_file=config_file)
+        init_endpoint_dir(name, endpoint_config=endpoint_config)
         print('''A default profile has been create for <{}> at {}
 Configure this file and try restarting with:
-    $ funcx-endpoint start {}'''.format(args.name,
-                                        os.path.join(endpoint_dir, 'config.py'),
-                                        args.name))
+    $ funcx-endpoint start {}'''.format(name,
+                                        new_config_file,
+                                        name))
         return
 
 
-def start_endpoint(args, global_config=None):
+@app.command(name="start")
+def start_endpoint(
+        name: str = typer.Argument("default", autocompletion=complete_endpoint_name),
+        endpoint_uuid: str = typer.Option(None, help="The UUID for the endpoint to register with")
+):
```

Comment:
Typer doesn't seem to do multiline helptext well. Here's a snippet from `funcx-endpoint --help`:
```
  restart    Restarts an endpoint
  start      Start an endpoint This function will do: 1.
  stop       Stops an endpoint using the pidfile
```
Also, it seems to mess up the ascii diagram. Which shouldn't be an issue.

Reply:
I think I will just explicitly set a short help message for that command, and the docstring, which is the default help text, can just be for other developers reading the code.  I wish that typer would follow the convention of simply taking the first line of the doc